tools/base/fpgaconf/fpgaconf.c

```diff
@@ -387,39 +298,56 @@ int parse_args(int argc, char *argv[])
 	}
 }
 
-/*
- * Check for bitstream header and fill out bistream_info fields
- */
-#define MAGIC 0x1d1f8680
-#define MAGIC_SIZE 4
-#define HEADER_SIZE 20
-int parse_metadata(struct bitstream_info *info)
+fpga_result get_fpga_interface_id(fpga_token token, fpga_guid interface_id)
 {
-	unsigned i = 0;
+	fpga_result result = FPGA_OK;
+	fpga_result resval = FPGA_OK;
+	fpga_properties filter = NULL;
+	fpga_objtype objtype;
+	fpga_guid guid;
+	errno_t e;
 
-	if (!info)
-		return -EINVAL;
+	result = fpgaGetProperties(token, &filter);
+	if (result != FPGA_OK) {
+		OPAE_ERR("Failed to get Token Properties Object \n");
```

Comment:
Doesn't this macro add a new line?

Reply:
Not related to this change set, but I fixed at any rate.